predicators/approaches/refinement_estimation_approach.py

```diff
@@ -34,8 +38,22 @@ def __init__(self,
                 CFG.sesame_max_skeletons_optimized), \
                "refinement_estimation_num_skeletons_generated should not be" \
                "greater than sesame_max_skeletons_optimized"
+        estimator_name = CFG.refinement_estimator
         self._refinement_estimator = create_refinement_estimator(
-            CFG.refinement_estimator)
+            estimator_name)
+        # If the refinement estimator is learning based, try to load
+        # trained state if the file exists.
```

Comment:
can we change the word "state" here to "model" or something like that to avoid confusion with `State`?